components/kernels/dotnet.js

```diff
@@ -33,7 +33,7 @@ export default () => (
             <SyntaxHighlighter language="zsh" style={github}>
               {installKernelSpecs}
             </SyntaxHighlighter>
-            The .NET kernel supports both C# and F# languages
+            The .NET kernel supports both C#, F# and Powershell languages
```

Comment:
"Both" typically means "two" so maybe change this to a list or replace "Both" with "the"

Reply:
Ditto that!

Reply:
addressing that .. wow :)